inst/python/sd2g.py

```
@@ -5,6 +5,7 @@
 from collections import defaultdict
 from time import perf_counter
 from xarray import DataArray
+from shapely.geometry import Polygon, Point
```

Ruff (0.11.9)

`shapely.geometry.Polygon` imported but unused

Remove unused import

(F401)